### PR TITLE
deps: update system-configuration to 0.6.0

### PR DESCRIPTION
**Pull request by saintEvol**
in the latest system-configuration, the IrDA interface types which will cause a link error on the latest ios has been deprecated.

### CODE REVIEW

**Comment by mxinden** [comment]
If you bump the patch version and add a changelog entry, I can cut a new release right after merge.

**Comment by saintEvol** [comment]
> If you bump the patch version and add a changelog entry, I can cut a new release right after merge.

Many thanks, I did bump the patch version and added a changelog entry

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
+## [3.2.1]
+
+### Fixed
+
+- Update `system-configuration` crates.
+  See [PR_37](https://github.com/mxinden/if-watch/pull/37)
```
Comment (mxinden) [inline CHANGELOG.md:13]:
```suggestion
- Update `system-configuration` crate.
  See [PR_37](https://github.com/mxinden/if-watch/pull/37).
```

**Review by mxinden — APPROVED** [review]

**Comment by mxinden** [comment]
@saintEvol is the CI failure related?

**Comment by saintEvol** [comment]
> @saintEvol is the CI failure related?

I checked again, it should be not related

**Comment by mangolas** [comment]
I stumbled on this problem as well, added #40 in a hope that system-configuration update could be released

**Comment by BlackbirdBop** [comment]
Hi, I'm facing the same issue. Hope it can be merged soon. Thanks for your great work!

**Comment by chanderlud** [comment]
Hello, I would appreciate it if this PR could be merged as the `libp2p` crate cannot compile for iOS until this is updated. Thanks!
@mxinden

**Comment by mxinden** [comment]
I am sorry for the delay here.

I have been maintaining `if-watch` for the [rust-libp2p](https://github.com/libp2p/rust-libp2p) project. I have recently [stepped down as a rust-libp2p maintainer](https://max-inden.de/post/2024-02-29-stepping-down/).

I will not maintain this crate going forward. I hope one of you is willing to take on maintainership through a fork.

Sorry for the trouble!

**Comment by jxs** [comment]
Hi @mxinden, sorry only saw this now, we can maintain it at Sigp, wdyt of just moving it there?

**Comment by mxinden** [comment]
@jxs works for me. Or the @libp2p org. Let me know what you prefer and I will attempt the move.

**Comment by jxs** [comment]
Hi @mxinden yeah libp2p also makes sense, thanks!

**Comment by mxinden** [comment]
Do you want it at `libp2p/if-watch` or `libp2p/rust-if-watch` @jxs?

**Comment by jxs** [comment]
Hi @mxinden , `libp2p/if-watch` should be enough, thanks!

**Comment by mxinden** [comment]
@jxs done: https://github.com/libp2p/if-watch

**Review by jxs — APPROVED** [review]
Thanks!

**Pull request merged**